src/omnisharp/server.ts

```diff
@@ -569,13 +570,34 @@ export class OmniSharpServer {
         return id;
     }
 
+    private static getLogLevelPrefix(logLevel: string) {
+        switch (logLevel) {
+            case "TRACE": return "trce";
+            case "DEBUG": return "dbug";
+            case "INFORMATION": return "info";
+            case "WARNING": return "warn";
+            case "ERROR": return "fail";
+            case "CRITICAL": return "crit";
+            default: throw new Error(`Unknown log level value: ${logLevel}`);
+        }
+    }
+
+    private _isFilterableOutput(logLevel: string, name: string, message: string) {
+        // filter messages like: /codecheck: 200 339ms
+        const timing200Pattern = /^\/[\/\w]+: 200 \d+ms/;
```

Comment:
Yeah, I completely agree. It already existed in the extension though (before I was on the project).